setup.py

```diff
@@ -38,6 +38,6 @@
             'xmltodict',
 
         ],
-        ':python_version < "3.5"': 'typing >= 3.5.3',
-    }
+    },
+    python_requires='>=3.5',
```

Comment:
```suggestion
    python_requires=">='3.5'",
```

Reply:
no the original was correct https://packaging.python.org/guides/distributing-packages-using-setuptools/#python-requires